cargo-dist/src/main.rs

```diff
@@ -516,3 +519,57 @@ fn cmd_manifest_schema(
     }
     Ok(())
 }
+
+async fn cmd_update(_config: &Cli, args: &cli::UpdateArgs) -> Result<(), miette::ErrReport> {
+    let mut updater = AxoUpdater::new_for("cargo-dist");
+    // Do we want to treat this as an error?
+    // Or do we want to sniff if this was a Homebrew installation?
+    if updater.load_receipt().is_err() {
+        eprintln!("Unable to load install receipt to check for updates.");
+        eprintln!("If you installed this via `brew`, please `brew upgrade cargo-dist`!");
```

Comment:
do we want to make this generic such as "via a package manager, like `brew`"

also- could we detect if it was installed via brew and report that? that'd be *nice*

Reply:
I think we can sniff that it was installed via brew and provide more detailed info, yeah.

Reply:
```
$ /usr/local/bin/cargo-dist dist update
Your copy of `cargo-dist` seems to have been installed via Homebrew.
Please run `brew upgrade cargo-dist` to update this copy.
```